checkoutmanager/tests/config.txt

```diff
@@ -153,3 +153,10 @@ We can ignore certain directories and support globbing to do so:
     >>> conf.report_missing(group='recipes') # doctest: +ELLIPSIS
     Unconfigured items in HOMEDIR/svn/recipes [svn]:
         HOMEDIR/svn/recipes/missing
+
+When used as a python module, DirInfo objects can be obtained programmatically:
+
+    >>> conf.directory_from_url('svn://svn/blablabla/trunk')
+    <DirInfo (svn) for HOMEDIR/svn/recipes/blablabla>
+    >>> conf.directory_from_path('HOMEDIR/svn/recipes/blablabla')
```

Comment:
The HOMEDIR should be handled differently. It is a parameter that's automatically converted by a regex when the doctest sees it in the output. It doesn't work in the input. Could you change it to this:

      >>> conf.directory_from_path('%s/svn/recipes/blablabla' % homedir)

That *ought* to get this test working.